models/prices/zkevm/prices_zkevm_tokens.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(
+        schema='prices_zkevm',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('weth-weth','zkevm','WETH',0x4f9a0e7fd2bf6067db6994cf12e4495df938e6e9,18),
+    ('reth-rocket-pool-eth','zkevm','RETH',0xb23c20efce6e24acca0cef9b7b7aa196b84ec942,18),
+    ('usdc-usdc','zkevm','USDC',0x37eaa0ef3549a5bb7d431be78a3d99bd360d19e5,6),
+    ('dai-dai','zkevm','DAI',0x744c5860ba161b5316f7e80d9ec415e2727e5bd5,18),
+    ('rseth-rseth','zkevm','RSETH',0x8c7d118b5c47a5bcbd47cc51789558b98dad17c5,18),
+    ('rwsteth-wrapped-liquid-staked-ether-20','zkevm','WSTETH',0x5d8cff95d7a57c0bf50b30b43c7cc0d52825d4a9,18),
```

Comment:
this one returns an error:
https://api.coinpaprika.com/v1/coins/rwsteth-wrapped-liquid-staked-ether-20

Reply:
i'll push a commit to remove